auto_cpufreq/bin/auto_cpufreq.py

```diff
@@ -134,17 +139,37 @@ def live_daemon_off():
                 tlp_service_detect()
             battery_setup()
             conf.notifier.start()
-            while True:
-                try:
-                    footer()
-                    gov_check()
-                    cpufreqctl()
-                    distro_info()
-                    sysinfo()
-                    set_autofreq()
-                    countdown(2)
-                except KeyboardInterrupt: break
-            conf.notifier.stop()
+            # while True:
```

Comment:
Let's make sure this code will be removed with new PR when old logic/code will be cleaned-up/removed.